DigitalLearningSolutions.Web/Attributes/RunValidateMethodAttribute.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace DigitalLearningSolutions.Web.Attributes
+{
+    using System.Collections.Generic;
+    using System.ComponentModel.DataAnnotations;
+    using System.Linq;
+
+    public class RunValidateMethodAttribute : ValidationAttribute

```

Comment:
This is a nice idea, but I'm worried it could be misused. I haven't yet seen a case where writing a validation method on a form data model is better than making a globally-usable attribute for it. Do you still think there are benefits to using this pattern?

Reply:
It may have some uses elsewhere but that is an unknown right now. 
There is a bit of a detachment here from the up-coming `HEEDLS-687` which was not considered during this refactor.
Given the actual target methods will become attributes this can go.

Reply:
This has gone.